explainer.md

```diff
@@ -1,624 +1,397 @@
-is [here](https://jsbin.com/niqiqididu/edit?html,output) (open in Firefox[^2]).
-
-[![Video Link for Shared Element Transition](https://img.youtube.com/vi/QzGEBUW-3U8/0.jpg)](https://youtu.be/QzGEBUW-3U8)
-
-## Animating Box Decoration CSS Properties
-
-A common capability desirable during transitions is to interpolate styles like
-border-radius that change the element's shape. The
-[container transform](https://material.io/design/motion/the-motion-system.html#container-transform)
-examples show a visual demo of that. Painting properties like the element's
-border within its image when using the element() function makes this difficult.
-
-Consider the same example as above with the addition of box decorations to
-the shared element.
-
-### Old Document
+- **transition part container**: If the part is created as a "container + child texture", this element will have a width and height of the content area of the original element, and have its computed styles reapplied. If the part is created as a "single texture", this element will have a width and height of the border box of the original element.
+- **transition part**: This element has a width and height of 100%, and `isolation: isolate`. This wrapper is useful when cross-fading textures (documented later).
+- **texture**: This contains the texture, which may paint outside the parent elements. This would behave like a replaced element, so would work with CSS properties like `object-fit`.
+  - Open question: How is painting outside handled? This element could overflow the parents in a developer-visible way, or we could avoid exposing that part somehow (as we currently do with paint overflowing).
+
+- Open question: What are these elements? Pseudo-elements are tricky to handle in some APIs (e.g., no `getBoundingClientRect` equivalent), and the nesting of pseudo-elements seems new. Alternatively, they could be `div`s in a shadow root, which means developers can use regular APIs on these elements.
+- Open question: How are styles applied to these elements? Inline styles are simple, but tricky to override in a stylesheet. An alternative would be to generate a `<style>` and put it, and the transition elements, in a shadow root.
+
+### Mixing in parts of Page-B and associating them with Page-A parts
+
+At this stage, Page-B identifies parts of its own page to be involved in the transition. This happens in the same way as part 1 with one difference: The textures and styles from Page-B will be updated if the underlying page updates. This means things like animated gifs will play, rather than being frozen on whatever frame they were on when they were captured.
+
+The developer can associate particular parts from Page-A to parts from Page-B. This would usually be done if they're equivalent. In this case, the headers, share buttons, and roots are equivalent. When this happens, the texture from the Page-B capture is added to the transition part:
```

Comment:
> This would usually be done if they're equivalent

Would we still support shared element becoming the root element?

Reply:
I'm not sure what you mean?

Reply:
The text here suggests that headers transition to headers, buttons to buttons, roots to roots. My question is do we support a div becoming a new root? Ie it's just a plain old element on page-A and it's the root element on page-B [eg](https://material.io/design/motion/the-motion-system.html#container-transform)